Brain/Brain.ino

```diff
@@ -116,10 +98,10 @@ void loop() {
 }
 
 bool write_to_SD(String message){
-  // every reading that we get from sensors should be written to sd and saved.
```

Comment:
Instead of sending a string; can we add error packets to the [sheet](https://docs.google.com/spreadsheets/d/1xNmjrb5a28FfVg-MG9jUBRiPyfdkjo7HiqPWP41N_IQ/edit#gid=998950668) and send that instead.